fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/patient/ChunkReader.java

```diff
@@ -118,12 +130,16 @@ public void open(Serializable checkpoint) throws Exception {
         adapter.registerRequestContext(ctx.getTenantId(), ctx.getDatastoreId(), ctx.getIncomingUrl());
 
         searchParametersForResoureTypes = BulkDataUtils.getSearchParametersFromTypeFilters(ctx.getFhirTypeFilters());
-        resourceType = ModelSupport.getResourceType(ctx.getFhirResourceType());
```

Comment:
This one was tricky to find!   `ctx.getFhirResourceType()` actually returns a comma-delimited list of resource types.   If you export a single type, it worked just fine.  If you exported multiple, then `ModelSupport.getResourceType(ctx.getFhirResourceType())` came back null.  Nasty!